tests/pytorch/test_numerics.py

```diff
@@ -67,6 +67,10 @@ def __init__(self, hidden_size, eps, num_attention_heads, embed, num_layers, seq
     "126m": ModelConfig(768, 1e-5, 12, 64, 12, 2048),
 }
 
+model_configs_small = {
+    "small": ModelConfig(128, 1e-5, 8, 36, 4, 128),
+}
```

Comment:
I see we're using an idiom of `model_configs | model_configs_small` throughout the code. It seems more natural if we just add `"small"` to `model_configs`:
```suggestion
    "126m": ModelConfig(768, 1e-5, 12, 64, 12, 2048),
    "small": ModelConfig(128, 1e-5, 8, 36, 4, 128),
```
Anyways, most of the tests shouldn't be iterating over all the model configs. We should explicitly pick the configs:
```python
@pytest.mark.parametrize("model", ["small"])
```

Reply:
I changed that. Only 3 models iterate through "small" and "126m".

Reply:
Ok, I need to add small model to more tests. To not increase number of tests I removed 126m run from *_accuracy test which needed to run small.